docs/api-reference/carto/carto-layer.md

```diff
@@ -2,17 +2,13 @@
 
 `CartoLayer` is the layer to visualize data using the CARTO Maps API.
 
-## Usage CARTO 3
+## Usage
 
 ```js
 import DeckGL from '@deck.gl/react';
 import {CartoLayer, setDefaultCredentials, MAP_TYPES, API_VERSIONS} from '@deck.gl/carto';
 
-setDefaultCredentials({
-  apiVersion: API_VERSIONS.V3
-  apiBaseUrl: 'https://gcp-us-east1.api.carto.com',
-  accessToken: 'XXX',
-});
+setDefaultCredentials({accessToken: 'XXX'});
```

Comment:
It might be useful to keep at least the apiBaseUrl prop because customers with self-hosted or SaaS in a different instance will need to change it.

Reply:
OK, I've added it back in marking as optional